stockfish/models.py

```diff
@@ -35,6 +35,25 @@ class Stockfish:
         "10.0": "2018-11-29",
     }
 
+    # The required type for each of the params, and any applicable min and max values:
```

Comment:
It might be good to link a source here so that it is transparent where the data comes from.

Reply:
Good idea, that makes sense to do - I'll link to the lines in ucioption.cpp.